modules/ipi-install-modifying-install-config-for-slaac-dual-stack-network.adoc

```diff
@@ -0,0 +1,52 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+:_content-type: PROCEDURE
+[id='ipi-install-modifying-install-config-for-slaac-dual-stack-network_{context}']
+= Optional: Configuring address generation modes for SLAAC in dual-stack networks
+
+For dual-stack clusters that use Stateless Address AutoConfiguration (SLAAC), you must specify a global value for the `ipv6.addr-gen-mode` network setting. You can set this value by configuring the ramdisk and the cluster configuration files. If you don't configure a consistent `ipv6.addr-gen-mode` in these locations, IPv6 address mismatches can occur between CSR resources and `BareMetalHost` resources in the cluster. 
+
+.Prerequisites
+
+* You have installed the `butane` utility.
+
+.Procedure
+
+. Create a Butane configuration file, `addr-more-nm-override.bu`, to define the `ipv6.addr-gen-mode` value:
++
+[source,yaml]
+----
+variant: openshift
+version: 4.13.0
+metadata:
+  name: 98-addr-mode-nm-override
+  labels:
+    machineconfiguration.openshift.io/role: master
+storage:
+  files:
+    - path: /etc/NetworkManager/system-connections/default.nmconnection
```

Comment:
I think this needs to be in /etc/NetworkManager/conf.d

The filename used in https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/commit/e6a33c04ebe1ac84e31628911e25bdfd7534dd3c is /etc/NetworkManager/conf.d/90-ipv6-addr-gen-mode-override.conf